tasks/stages/update-website

```diff
@@ -16,9 +16,15 @@ git clone --depth=1 "https://$BPMN_IO_TOKEN@github.com/bpmn-io/bpmn.io.git" "$CL
 
 cd "$CLONE_DIR"
 
-echo "Updating toolkit version to $TAG on bpmn.io"
+PUBLISHED=`date +"%F %H:%M"`
+
+echo "Updating toolkit version to version=$TAG, published=$PUBLISHED on bpmn.io"
+
+cat src/data/site.yml | \
+  tr "\r?\n" "\r" | \
+  sed -e "s#dmnjs:\r  version: [^\r]*\r  published: [^\r]*\r#dmnjs:\r  version: $TAG\r  published: $PUBLISHED\r#" | \
+  tr "\r" "\n" > src/data/site.yml_new
```

Comment:
I tried this out locally and it swallowed cmmnjs key:

```yml
title: bpmn.io
bpmnjs:
  version: v9.0.3
  published: 2022-03-14 13:46
  changelog: https://github.com/bpmn-io/bpmn-js/blob/develop/CHANGELOG.md
dmnjs:
  version: lol
  published: 2022-04-11 15:00
  version: v0.19.2
  published: 2019-08-11 14:30
  changelog: https://github.com/bpmn-io/cmmn-js/blob/develop/CHANGELOG.md
formjs:
  version: v0.7.1
  published: 2022-02-21 10:49
  changelog: https://github.com/bpmn-io/form-js/blob/master/packages/form-js/CHANGELOG.md
```

Reply:
It worked when I replaced `published: [^\r]*\r` with `published: [^\n]*\n` and `published: $PUBLISHED\n`.

Reply:
Are you sure `tr` works on MacOS?

Because the script works on Linux (our CI system).

Reply:
Trust me :parrot: :hear_no_evil: 

![image](https://user-images.githubusercontent.com/58601/162759563-9a7cfec4-5880-46e5-b352-ddc4c067267f.png)

Reply:
If it works on Linux then I'm OK with that.

Reply:
I don't get though why we have `\r` in the first place.